bundles/org.eclipse.equinox.servletbridge/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: %bundleName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.equinox.servletbridge;singleton:=true
-Bundle-Version: 1.6.200.qualifier
+Bundle-Version: 1.6.300.qualifier
```

Comment:
I don't see any changes to the servletbridge.

Reply:
the build complained about it

Reply:
removed it